src/v/security/tests/role_store_bench.cc

```diff
@@ -26,14 +28,30 @@ using role_member = security::role_member;
 using role_member_type = security::role_member_type;
 using role = security::role;
 using role_store = security::role_store;
+using authorizer = security::authorizer;
```

Comment:
Might as well bite the bullet and  `using namespace security;`

Reply:
yeah, fair enough. this is sicko behavior

Reply:
done